.github/workflows/checkstyle.yaml

```diff
@@ -13,32 +13,44 @@ jobs:
         ref: ${{ github.event.pull_request.head.sha }}
     - name: Install dependencies
       run: |
+        echo 'force-unsafe-io' | sudo tee /etc/dpkg/dpkg.cfg.d/force_unsafe_io
```

Comment:
Maybe just use something like https://github.com/abbbi/github-actions-tune ? Looks more thorough than manual commands.

Reply:
We may look at it in different PR.

Reply:
Thanks for pointing to this article. Looks like we might incorporate those ideas in here. I made couple of runs and the results are promising.

Reply:
@szubersk  Just wanted to note, that I've thought of using https://github.com/abbbi/github-actions-tune github action itself, or suitable analogs, if we can do so. I'd prefer to reuse and contribute to something common.

Reply:
I reviewed that project and honestly I am skeptical if it is mature enough to rely on it. It has a couple of good ideas which can be condensed to 4 shell code lines though.